config/checkstyle-exclude.xml

```diff
@@ -62,6 +62,7 @@
 
     <!--DBCollection is insanely long, and we should not compromise every class length for this one-->
     <suppress checks="FileLength" files="DBCollection"/>
+    <suppress checks="FileLength" files="MongoCollection"/>
```

Comment:
The file MongoCollection.java exceeds 2000 lines with the additions to support updates with pipelines, which generates a checkstyle error. I did not find a way to use a Java annotation, so I suppressed the error here.

Reply:
Ack